controllers/perconaserverformysql_controller.go

```diff
@@ -215,12 +224,18 @@ func (r *PerconaServerForMySQLReconciler) reconcileOrchestrator(
 	ctx context.Context,
 	cr *apiv2.PerconaServerForMySQL,
 ) error {
-	err := k8s.EnsureObjectWithHash(ctx, r.Client, cr, orchestrator.StatefulSet(cr), r.Scheme)
-	if err != nil {
+	l := log.FromContext(ctx).WithName("reconcileOrchestrator")
+
+	if cr.MySQLSpec().Size < 2 {
+		l.Info("not enough mysql replicas. skip", "size", cr.MySQLSpec().Size)
+		return nil
+	}
```

Comment:
what if cluster started with Size=3 and later changed to Size=1

Reply:
removed this check https://github.com/percona/percona-server-mysql-operator/pull/21/commits/f16ed3c50fa99b48e0f6b0d8cbb2de49e4f3c6d6